src/decisions/admin/helpers/parse-decision-value.helper.ts

```
@@ -0,0 +1,9 @@
+export function parseDecisionValue(value: string): number | null {
+  const result = parseInt(value, 10);
+
+  if (isNaN(result) || result < 0) {
```

I've just tested this out locally - I think this should probably submit 0 instead of an empty string. We can fix that as part of a separate PR though, if you think it's worth doing?

---

The idea was to make a distinction between "0" and a value that hasn't been filled in yet. It feels dangerous to silently replace an invalid value with a valid one?

---

Probably something we can do a bit of further testing on if necessary - let's leave it as is for now